Makefile

```diff
@@ -13,6 +13,9 @@ flake8:
 	flake8
 
 example:
+	python example/manage.py migrate --noinput
+	-DJANGO_SUPERUSER_PASSWORD=p python example/manage.py createsuperuser \
+		--noinput --username=$(USER) --email=$(USER)@mailinator.com
```

Comment:
That looks very useful! Sholdn't this be `"$(USER)"` though, for safety? OTOH people will only hack themselves 😅

Reply:
Good point :+1:. I added quotes in the latest revision.